.github/workflows/duplication.yml

```diff
@@ -0,0 +1,28 @@
+name: duplicate-test-check
+on:
+  pull_request:
+permissions:
+  actions: read
+jobs:
+   run_tests_1:
+     runs-on: ubuntu-latest
+     steps:
+       - name: Checkout Ivy 🛎
+         uses: actions/checkout@v2
+         with:
+           path: ivy
+           persist-credentials: false
+           submodules: "recursive"
+           fetch-depth: 100
+
+
+       - name: Run Tests
+         id: tests
+         run: |
+           cd ivy
+           docker run --rm -v pwd:/ivy -v pwd/.hypothesis:/.hypothesis unifyai/ivy:latest python3 duplicate.py
```

Comment:
Hi @zaeemansari70, please use the same exact command as mentioned below -:

`docker run --rm -v "$(pwd)":/ivy -v "$(pwd)"/.hypothesis:/.hypothesis unifyai/ivy:latest python3 duplicate.py`